package.json

```diff
@@ -58,15 +59,9 @@
   "resolutions": {
     "**/tslib": "^2.3.1"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
   "lint-staged": {
     "*.{ts,tsx,js,jsx,json,md,mdx}": [
-      "prettier --write",
-      "git add"
```

Comment:
Manual add not needed anymore since v10:

https://github.com/okonet/lint-staged/releases/tag/v10.0.0

> Prior to version 10, tasks had to manually include git add as the final step. This behavior has been integrated into lint-staged itself in order to prevent race conditions with multiple tasks editing the same files. If lint-staged detects git add in task configurations, it will show a warning in the console. Please remove git add from your configuration after upgrading.

Reply:
Good catch! Forgot about that